src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -290,12 +322,17 @@ def _convert_hms_table_to_external(self, src_table: Table):
                 old_table.viewOriginalText(),
             )
             self._catalog.alterTable(new_table)
+            self._update_table_status(src_table, inventory_table)
             logger.info(f"Converted {src_table.name} to External Table type.")
         except Exception as e:  # pylint: disable=broad-exception-caught
             logger.warning(f"Error converting HMS table {src_table.name} to external: {e}", exc_info=True)
             return False
         return True
 
+    def _update_table_status(self, src_table: Table, inventory_table: str):
+        update_sql = f"update {escape_sql_identifier(inventory_table)} set object_type = 'EXTERNAL' where catalog='hive_metastore' and database='{src_table.database}' and name='{src_table.name}';"
```

Comment:
```suggestion
    def _update_table_status(self, src_table: Table, inventory_table: str):
        update_sql = f"UPDATE {self._tc.full_name} SET object_type = 'EXTERNAL' WHERE catalog='hive_metastore' AND database='{src_table.database}' AND name='{src_table.name}';"
```
1. remove `inventory_table` argument from the call stack - we already have the table crawler injected to this class
2. nit: SQL keywords have to be uppercase